client/src/components/StepWizard/StyledTextInput.js

```diff
@@ -17,14 +18,35 @@ const StyledTextInput = styled(InputItem)`
   .am-input-control {
     input {
       ${theme.form.input}
-      color: ${theme.colors.darkGray};
-      border: 0.1rem solid ${theme.colors.primary};
-      border-width: 0.1rem;
-      border-radius: 5px;
+      color: ${darkGray};
+      border: none;
+      border-bottom: 0.1rem solid ${royalBlue} !important;
```

Comment:
Yea, I agree. There was some discussion about using `!important` because some styles were to override ant-design styles

Reply:
I only used important because I couldn't find another way to overwrite ant-design... I'll try again

Reply:
It you can't its fine, I didn't know the context of it so don't worry about it!

Reply:
I can't figure out why the StyledTextInput for the email address doesn't display the border correctly... Any ideas? It should be 2px wide all around, but it looks much thinner on the bottom

Reply:
Never mind I found it :)

Reply:
> Should avoid !important here since you already declared `border: none; ` above. This should be in mobile only because it looks weird in desktop - attached screenshot in comments.

Because you overrode it with `!important`. If you really need `!important`, you should put in mobile's media query :)